devdocs/process.md

```diff
@@ -111,7 +111,8 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      If you did not get a review within 2-3 days, it is OK to request for a review 
      by posting a comment in the PR. 
 
-   * Once the PR is open, try and complete it within 2 weeks. Inactivity for a longer period would necessitate a restart of the PR.
+   * Once the PR is open, try and complete it within 2 weeks. Inactivity for a 
+     longer period would necessitate a restart of the PR.
```

Comment:
sir I dont think this makes any difference to the output file. Did you intend this modification?